vendor/assets/javascripts/behaviors/nested_resources.js

```diff
@@ -135,9 +135,15 @@ Cmsify.NestedResource.prototype.deserializeFields = function(fields) {
 };
 
 Cmsify.NestedResource.prototype.setIsModified = function () {
-  if (tinyMCE.editors.some(function(editor) { return editor.isDirty() }) ||
-      JSON.stringify(this.serializeFields()) != JSON.stringify(this.lastData)) {
+  var tinyMCEDirty = false;
+  tinyMCE.editors.some(function(editor) {
+    if (editor.isDirty()) tinyMCEDirty = true;
+  }.bind(this));
+
+  if (JSON.stringify(this.serializeFields()).localeCompare(JSON.stringify(this.lastData)) != 0) {
```

Comment:
@toastynerd - I don't see how any of this tinyMCE functionality is changing. Can you explain exactly what this PR is changing and why it fixes the issue?